src/nci_adapter.c

```diff
@@ -430,7 +554,8 @@ nci_adapter_nci_intf_activated(
             tag = nci_adapter_create_known_tag(self, ntf);
         }
         if (!tag) {
-            nfc_adapter_add_other_tag(NFC_ADAPTER(self), self->target);
+            nfc_adapter_add_other_tag2(NFC_ADAPTER(self), self->target,
+                nfc_tag_param(tag));
```

Comment:
Wait, but if `tag` is NULL then wouldn't `nfc_tag_param(NULL)` just always return `NULL` and that's it? The point of the deprecation warning was that poll parameters should still be provided whenever possible, even if the exact type of tag is unknown or unsupported.

Reply:
I have changed that logic, please review, thanks